scripts/setup.sh

```
@@ -0,0 +1,236 @@
+    docker run -d --name nixopus-db \
+        -e POSTGRES_USER="${POSTGRES_USER:-nixopus}" \
+        -e POSTGRES_PASSWORD="${POSTGRES_PASSWORD:-nixopus}" \
+        -e POSTGRES_DB="${POSTGRES_DB:-nixopus}" \
+        -p "${POSTGRES_PORT:-5432}:5432" \
+        postgres
+    echo "Postgres setup completed successfully"
+}
+
+# setup ssh will create a ssh key and add it to the authorized_keys file
+function setup_ssh(){
+    continue
```

Shellcheck (0.10.0)

[error] In functions, use return instead of continue.

(SC2104)